torch/_custom_ops.py

```diff
@@ -250,7 +250,7 @@ def impl_abstract(qualname, *, func=None):
     """
     import torch.library
 
-    return torch.library.impl_abstract(qualname, func, _stacklevel=2)
+    return torch.library.register_fake(qualname, func, _stacklevel=2)
```

Comment:
The deprecation is intentional: we shouldn't be using anything in torch._custom_ops anymore.